blocks.go

```diff
@@ -104,3 +104,14 @@ func (b *Blocks) GetByIndex(ix uint64) (*Block, error) {
 
 	return block, nil
 }
+
+func (b *Blocks) GetAll() []*Block {
```

Comment:
Care to rename to `Clone()` and use `copy(blocks, b.buffer)` instead?

Reply:
It doesn't clone actual structure, just returns blocks store.
Not sure whats the point in copying buffer, those blocks aren't modified and won't be modified by anything else.

Reply:
The two should be equivalent; `copy` would just copy the pointers rather than the actual `Block` value itself.

Reply:
yeah, so copy does same as append. i can change if you insist, but i see no reasons why.
what about name?

Reply:
For the name, I reckon `Clone` would be better as we are cloning the buffer itself (as a slice of pointers). `copy` is just a little more idiomatic in this context so that we know we're taking a copy of the buffer.

Reply:
I guess it makes sense then to update style guide with preference of `copy` over `append`